### PR TITLE
Microsoft Defender for Cloud (Azure security center) event hub

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10533

## Description
This PR adds mapping to Microsoft Defender for Cloud  that are ingested by Azure event hub

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AzureSecurityCenter/README.md [hunk 1]
```diff
@@ -11,3 +11,64 @@ Azure Security Center provides threat protection for data centers within both cl
...
+<~XSIAM>
+
+### This pack includes:
+- Log Normalization - XDM mapping.
+
+### Supported Event Types:
+- Security Alerts
+
+### Supported Timestamp Formats:
+- MMM DD YYYY HH:MM:SS (UTC)
```
Comment (ShirleyDenkberg) [inline Packs/AzureSecurityCenter/README.md:24]:
```suggestion
Log Normalization - XDM mapping.

### Supported Event Types:
Security Alerts

### Supported Timestamp Formats:
MMM DD YYYY HH:MM:SS (UTC)

```

File: Packs/AzureSecurityCenter/README.md [hunk 2]
```diff
@@ -11,3 +11,64 @@ Azure Security Center provides threat protection for data centers within both cl
...
+
+Cortex XSIAM supports two methods to fetch alerts from Microsoft Defender for Cloud:
+- Microsoft Defender for Cloud collector (API Collection).
+- Azure Event Hub.
+
+### Microsoft Defender for Cloud collector 
+
+This collection method is more relevant in cases where the number of subscriptions is low, as each integration's instance refers to one subscription.
+
+For more information on how to configure this integration, please refer to the integration's docs:
```
Comment (ShirleyDenkberg) [inline Packs/AzureSecurityCenter/README.md:38]:
```suggestion
For more information on how to configure this integration, refer to the integration's docs:

```
I think it makes sense to move this to after the instructions for creating an instance. WDYT?

File: Packs/AzureSecurityCenter/README.md [hunk 3]
```diff
@@ -11,3 +11,64 @@ Azure Security Center provides threat protection for data centers within both cl
...
+1. navigate to **settings** &rarr; **Automation & Feed Integrations**.
```
Comment (ShirleyDenkberg) [inline Packs/AzureSecurityCenter/README.md:39]:
```suggestion
1. Navigate to **settings** &rarr; **Automation & Feed Integrations**.

```

File: Packs/AzureSecurityCenter/README.md [hunk 4]
```diff
@@ -11,3 +11,64 @@ Azure Security Center provides threat protection for data centers within both cl
...
+2. Search for **Microsoft Defender for Cloud**.
+3. Click on **+ Add instance**.
```
Comment (ShirleyDenkberg) [inline Packs/AzureSecurityCenter/README.md:41]:
```suggestion
3. Click **+ Add instance**.

```

File: Packs/AzureSecurityCenter/README.md [hunk 5]
```diff
@@ -11,3 +11,64 @@ Azure Security Center provides threat protection for data centers within both cl
...
+This collection method is more relevant in cases where the number of subscriptions is high, and you want to stream alerts at the tenant level.
+Nevertheless, it supports alerts streaming at the subscription level as well.
+
+#### Prerequisites
+- Create an **Azure event hub**. For more information, refer to Microsoft's official [documentation](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
+- Make sure that you have permissions for the root management group.
+
+#### Stream alerts with Continuous Export to Event Hub
+
+Refer to the below links for detailed instructions:
```
Comment (ShirleyDenkberg) [inline Packs/AzureSecurityCenter/README.md:53]:
```suggestion
Refer to the following links for detailed instructions:

```

**Comment by ShirleyDenkberg** [comment]
@OBavly Doc review completed.

**Review by OBavly — APPROVED** [review]

**Pull request merged**